src/Microsoft.AspNetCore.Mvc.Razor.Host/MvcTagHelperAttributeValueCodeRenderer.cs

```diff
@@ -63,7 +63,11 @@ public MvcTagHelperAttributeValueCodeRenderer(GeneratedTagHelperAttributeContext
             if (attributeDescriptor.TypeName.Equals(_context.ModelExpressionTypeName, StringComparison.Ordinal))
             {
                 writer
-                    .WriteStartMethodInvocation(_context.CreateModelExpressionMethodName)
+                    .WriteStartInstanceMethodInvocation(
+                    _context.ModelExpressionProviderPropertyName,
+                    _context.CreateModelExpressionMethodName)
```

Comment:
Either move the arguments onto the same line as the `WriteStartInstanceMethodInvocation()` method (if it fits) or indent them another 4 spaces. Not readable as-is.